### PR TITLE
fix(config): correct heatit brand names

### LINKED ISSUE #1768: Heatit Z-Smoke Detector  - missing device configuration
Heatit Z-Smoke Detector 230V

Manufacturer ID: 0x019B 
Product ID: 0x3601 
Product Type ID: 0x0003   

- [ x] It is not in the [configuration DB](https://devices.zwave-js.io/)
- [x ] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)
- [ x] It is not in the [TODO list](https://github.com/zwave-js/node-zwave-js/issues/1600)

PDF Manual - https://media.heatit.com/709

https://products.z-wavealliance.org/products/3933?selectedFrequencyId=1
https://products.z-wavealliance.org/Products/3933/XML

I have no prior knowledge of how to add device configurations, I'm happy to help if there are docs i can follow.

### PR DESCRIPTION
**Pull request by blhoward2**
Closes #1768. Fixes Heatit brand names in some files, cleans up values.

### CODE REVIEW

File: packages/config/config/devices/0x019b/4512689.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
-// ThermoFloor 4512689
-// Heatit Z-Smoke Detector 230V
+// Heatit 4512689
+// Z-Smoke Detector 230V
 {
-	"manufacturer": "ThermoFloor",
+	"manufacturer": "THeatit",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/4512689.json]:
```suggestion
	"manufacturer": "Heatit",
```

File: packages/config/config/devices/0x019b/4512688.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
-// ThermoFloor 4512688
-// Heatit Z-Smoke Battery
+// Heatit 4512688
+// Z-Smoke Battery
 {
-	"manufacturer": "ThermoFloor",
+	"manufacturer": "Heatit",
 	"manufacturerId": "0x019b",
 	"label": "4512688",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/4512688.json]: I'd put Z-Smoke and similar in the labels

File: packages/config/config/devices/0x019b/heatit_z-relay.json [hunk 3]
```diff
@@ -41,31 +57,51 @@
 			"allowManualEntry": true
 		},
 		"3": {
-			"label": "Thermistor type for input 1.",
-			"description": "Configures the thermistor type connected to input 1.",
+			"label": "Thermistor Type Input 1",
+			"description": "This parameter decides which kind of thermistor that is connected to the input..",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/heatit_z-relay.json]:
```suggestion
			"description": "This parameter decides which kind of thermistor is connected to input 1.",
```

File: packages/config/config/devices/0x019b/heatit_z-relay.json [hunk 4]
```diff
@@ -41,31 +57,51 @@
...
+					"label": "10K NTC",
+					"value": 1
+				}
+			]
 		},
 		"4": {
-			"label": "Thermistor type for input 2.",
-			"description": "Configures the thermistor type connected to input 2. VALUE DESCRIPTION",
+			"label": "Thermistor Type Input 2",
+			"description": "This parameter decides which kind of thermistor that is connected to the input.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/heatit_z-relay.json]:
```suggestion
			"description": "This parameter decides which kind of thermistor is connected to input 2.",
```

File: packages/config/config/devices/0x019b/heatit_z-relay.json [hunk 5]
```diff
@@ -142,29 +177,57 @@
 			"allowManualEntry": true
 		},
 		"12": {
-			"label": "Auto relay on.",
-			"description": "Configures internal states that will cause the relay to be turned on.",
+			"label": "Auto Relay On",
+			"description": "Configures the state that will cause the relay to be turned on",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/heatit_z-relay.json]:
```suggestion
			"description": "Configures which events will cause the relay to be turned on",
```

File: packages/config/config/devices/0x019b/heatit_z-relay.json [hunk 6]
```diff
@@ -142,29 +177,57 @@
...
+					"label": "No flood detected",
+					"value": 1
+				}
+			]
 		},
 		"13": {
-			"label": "Auto relay off.",
-			"description": "Configures internal states that will cause the relay to be turned off.",
+			"label": "Auto Relay Off",
+			"description": "Configures the state that will cause the relay to be turned off",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/heatit_z-relay.json]:
```suggestion
			"description": "Configures which events will cause the relay to be turned off",
```

File: packages/config/config/devices/0x019b/heatit_z_push_button_8.json [hunk 7]
```diff
@@ -1,9 +1,9 @@
-// ThermoFloor Heatit Z Push Button 8
+// Heatit Z Push Button 8
 // Wall Mounted Switch
 {
-	"manufacturer": "ThermoFloor",
+	"manufacturer": "Heatit",
 	"manufacturerId": "0x019b",
-	"label": "Heatit Z Push Button 8",
+	"label": " Z Push Button 8",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/heatit_z_push_button_8.json]:
```suggestion
	"label": "Z Push Button 8",
```

File: packages/config/config/devices/0x019b/heatit_z_water.json [hunk 8]
```diff
@@ -58,7 +57,7 @@
 			"allowManualEntry": true
 		},
 		"3": {
-			"label": "Thermistor type input 1",
+			"label": "Thermistor Type Input 1",
 			"description": "This parameter decides which kind of thermistor that is connected to the input..",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/heatit_z_water.json:61]: see previous file (same for the next labels)

File: packages/config/config/devices/0x019b/heatit_z_water.json [hunk 9]
```diff
@@ -182,30 +185,22 @@
 			"allowManualEntry": true
 		},
 		"11": {
-			"label": "Input 1 report interval",
+			"label": "Input 1 Report Interval",
 			"description": "Time interval between consecutive temperature reports.",
 			"valueSize": 2,
+			"unit": "seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/heatit_z_water.json]:
```suggestion
			"unit": "10 seconds",
```

Comment (AlCalzone) [reply packages/config/config/devices/0x019b/heatit_z_water.json]: when I go by the option description. Might affect all these parameters

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
ready?

**Comment by blhoward2** [comment]
Should be

**Pull request merged**